lib/LLVMIRCodeGen/BundleSaver.cpp

```diff
@@ -450,6 +457,66 @@ void BundleSaver::emitSymbolTable() {
                            irgen_->getBundleName() + "SymbolTable");
 }
 
+void BundleSaver::addBundleObjects(
```

Comment:
The name `addBundleObjects` does not quite reflect what you do here, IMHO. You create an archive/static library. Maybe the name should be "createBundleArchive" or something like that?

Reply:
Renamed it to `createBundleArchive` as you suggested.